workers/ohsome_quality_analyst/indicators/building_completeness/indicator.py

```diff
@@ -98,7 +98,11 @@ async def preprocess(self) -> None:
             item["result"][0]["value"] for item in query_results["groupByResult"]
         ]
         # Get covariates (input parameters or X)
-        shdi = await db_client.get_shdi(hex_cells)
+        # Set default SHDI val to 0 for cells not intersecting with SHDI geometry
+        shdi = [{"sum": 0} for feature in hex_cells.features]
```

Comment:
do we use `sum` for SHDI, I thought it would be `mean` or something similar?

The default value should not be 0, but something more realistic? The minimum SHDI for a country is about 0.3 or 0.4

Reply:
Renamed the result field back to just `shdi`